lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -32,20 +32,21 @@ static void RecyclerExecuteICUWithRetry(_In_ ICUFunc func, _In_ Recycler *recycl
     AssertOrFailFastMsg(success, "Could not allocate buffer for ICU call");
 }
 
-#define ICU_ERROR_FMT _u("INTL: %S failed with error code %S\n")
-#define ICU_EXPR_FMT _u("INTL: %S failed expression check %S\n")
+#if defined(DBG) || defined(ENABLE_DEBUG_CONFIG_OPTIONS)
+#define INTL_TRACE(fmt, ...) Output::Trace(Js::IntlPhase, _u("%S(): " fmt "\n"), __func__, __VA_ARGS__)
+#else
+#define INTL_TRACE(fmt, ...)
+#endif
 
 #define ICU_ASSERT(e, expr)                                                   \
     do                                                                        \
     {                                                                         \
         if (ICU_FAILURE(e))                                                   \
         {                                                                     \
-            ICU_DEBUG_PRINT(ICU_ERROR_FMT, ICU_ERRORMESSAGE(e));              \
```

Comment:
>ICU_DEBUG_PRINT(ICU_ERROR_FMT, ICU_ERRORMESSAGE(e));     [](start = 12, length = 56)

@jackhorton not replacing with trace?

Reply:
We are about to AssertOrFailFast anyways, so they wouldn't give too much additional information